examples/python/python3/HistoricalDataRequest.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# usage: python3 HistoricalDataRequest.py <host-ip>
+
+import argparse
+import json
+import ssl
+import sys
+import urllib.request
```

Comment:
I didn't have Python3 installed, but looking at the docs for `urllib`, it looks like they broke out this module into smaller sub-modules.  Thanks for taking the time for double-check and verify this.